app/assets/components/elements/json-delete.html

```diff
@@ -1,9 +1,18 @@
-<polymer-element name=json-delete extends=json-req>
-
-    <script>
-        Polymer({
-            method: 'delete'
-        });
-    </script>
-
-</polymer-element>
+<!--
+    TODO(polyup): Inheriting from other custom elements is not yet supported.
+    See: https://www.polymer-project.org/1.0/docs/migration.html#inheritance
```

Comment:
I assume you can replace inheritance with mix-ins (or whatever polymer calls them)